Sources/Helpers/OAAppSettings.h

```diff
@@ -970,10 +970,13 @@ typedef NS_ENUM(NSInteger, EOARateUsState)
 // Widgets
 
 @property (nonatomic) OACommonListOfStringList *leftWidgetPanelOrder;
-@property (nonatomic) OACommonListOfStringList *topWidgetPanelOrder;
 @property (nonatomic) OACommonListOfStringList *rightWidgetPanelOrder;
+@property (nonatomic) OACommonListOfStringList *topWidgetPanelOrder;
 @property (nonatomic) OACommonListOfStringList *bottomWidgetPanelOrder;
 
+@property (nonatomic) OACommonListOfStringList *topWidgetPanelOrderOld;
+@property (nonatomic) OACommonListOfStringList *bottomWidgetPanelOrderOld;
```

Comment:
Can we refrain from using the new properties topWidgetPanelOrderOld and bottomWidgetPanelOrderOld and instead override the old topWidgetPanelOrder/bottomWidgetPanelOrder in the changeWidgetIdsMigration1 function?